samples/sample13/Ballerina.toml

```diff
@@ -2,3 +2,12 @@
 org-name = "kathy"
 version = "0.0.1"
 
+[platform]
+target = "java8"
+
+    [[platform.libraries]]
+    artifactId = "toml4j"
+    version = "0.4.0"
+    path = "./libs/mysql-connector-java-8.0.11.jar"
+    groupId = "com.moandjiezana.toml"
+    module = "toml4j"
```

Comment:
This is mysql jar, not toml. Find the correct one from maven central.

Reply:
Updated the .toml